### PR TITLE
[Wallet] Show a message on the verification screen to encourage not closing the app

### PR DESCRIPTION
**Pull request by etuleu**
### Description

Just a message to encourage people to keep the app open during the verification process.

### Other changes

Created a component that can alternate between two pieces of text.
Could add animation or fade in and out but want to get this in before tomorrow first.

### Tested

Manually.

### Related issues

- Short term solution until we can land #5142

### Backwards compatibility

Yes.

### CODE REVIEW

File: packages/mobile/src/verify/AlternatingText.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const SHOW_TIME = 2 * 1000 // ms
+
+interface Props {
+  style: any
+  primaryText: string
+  secondaryText: string
+}
+
+export default function AlternatingText({ style, primaryText, secondaryText }: Props) {
+  const [time, setTimer] = useState(0)
```
Comment (jeanregisser) [inline packages/mobile/src/verify/AlternatingText.tsx]: From what I see, this state could be removed.

File: packages/mobile/src/verify/AlternatingText.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const [isPrimary, setIsPrimary] = useState(true)
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimer((t) => t + 1)
+      setIsPrimary(time % 2 === 0)
+    }, SHOW_TIME)
+
+    return () => clearInterval(interval)
+  }, [time, isPrimary])
```
Comment (jeanregisser) [inline packages/mobile/src/verify/AlternatingText.tsx]:
```suggestion
  useEffect(() => {
    const interval = setInterval(() => {
      // Quick and easy way to animate the state change
      LayoutAnimation.easeInEaseOut()
      setIsPrimary((wasPrimary) => !wasPrimary)
    }, SHOW_TIME)

    return () => clearInterval(interval)
  }, [])
```

File: packages/mobile/src/verify/AlternatingText.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react'
+import { Text, View } from 'react-native'
+
+// how often to swap the text
+const SHOW_TIME = 2 * 1000 // ms
+
+interface Props {
+  style: any
```
Comment (jeanregisser) [inline packages/mobile/src/verify/AlternatingText.tsx]:
```suggestion
  style: StyleProp<TextStyle>
```

File: packages/mobile/src/verify/AlternatingText.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    }, SHOW_TIME)
+
+    return () => clearInterval(interval)
+  }, [time, isPrimary])
+
+  return (
+    <View>
+      {isPrimary && <Text style={style}>{primaryText}</Text>}
+      {!isPrimary && <Text style={style}>{secondaryText}</Text>}
+    </View>
```
Comment (jeanregisser) [inline packages/mobile/src/verify/AlternatingText.tsx]:
```suggestion
    <Text style={style}>{isPrimary ? primaryText : secondaryText}</Text>
```

**Review by jeanregisser — CHANGES_REQUESTED** [review]
Thanks! 👍

File: packages/mobile/src/verify/AlternatingText.tsx (hunk 1 above)
Comment (etuleu) [reply packages/mobile/src/verify/AlternatingText.tsx]: much better, thanks @jeanregisser

**Review by jeanregisser — APPROVED** [review]
LGTM! 👍

**Comment by Lss-Ankit** [comment]
@jeanregisser Verified with Test flight build v1.4.0 (28) and Android play store (internal build) v1.4.0 (1004294317) and found the following :
- "Confirming..." & "Please Keep the app open" message is shown wise versa when user is verifying phone number screen. 

Please let us know if we need to verify anything else ? 
Thanks!

**Pull request merged**